### PR TITLE
[Zip][Portable] New Package: Blinue.Magpie 0.9.1

### PR DESCRIPTION
**Pull request by Trenly**
* Resolves #32734

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/88063)

### CODE REVIEW

**Comment by Trenly** [comment]
@msftbot zip

**Review by ItzLevvie — DISMISSED** [review]

**Comment by Trenly** [comment]
Ping-pong @stephengillie

File: manifests/b/Blinue/Magpie/0.9.1/Blinue.Magpie.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.1.6 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.4.0/manifest.installer.1.4.0.json
+
+PackageIdentifier: Blinue.Magpie
+PackageVersion: 0.9.1
+ReleaseDate: 2022-07-25
+InstallerType: zip
+Installers:
```
Comment (stephengillie) [inline manifests/b/Blinue/Magpie/0.9.1/Blinue.Magpie.installer.yaml:8]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
Installers:
```

Comment (stephengillie) [reply manifests/b/Blinue/Magpie/0.9.1/Blinue.Magpie.installer.yaml:8]: Hi @Trenly,

The package requires this dependency to run:
image

However, it gives a different message with the dependency installed:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**